rolling/distribution.yaml

```diff
@@ -2699,6 +2699,17 @@ repositories:
       url: https://github.com/iRobotEducation/irobot_create_msgs.git
       version: rolling
     status: developed
+  jacro:
+    release:
```

Comment:
Do you want to include a `doc` tag so that documentation is automatically generated for your package? The documentation will be accessible from your package page on index.ros.org.